README.md

```
@@ -287,3 +287,88 @@ public void handleOrder(@Payload OrderCreatedEvent event, @Header("message_id")
+TokenPair tokenPair = tokenProvider.createTokenPair(userDetails);
+
+String accessToken = tokenPair.getAccessToken();   // "Bearer " 접두사 포함
+String refreshToken = tokenPair.getRefreshToken();
+Long expires = tokenPair.getAccessTokenExpiresIn();
+```
+
+### 토큰 검증 및 정보 추출
+
+* 정보 추출 기능(parseClaims() 메서드)의 경우, 해당 기능을 사용할 서비스의 build.gradle에 jjwt-api 의존성을 추가해야 사용 가능
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)